### ISSUE TITLE
Reset settings to default

### ISSUE THREAD

**Issue by surak** [ccrisan/motionpie#343]
I can't make motion detection work, my camera has the leds all on, and I have no idea why. I would like to reset everything instead of reinstalling it from the image.

**Comment by ccrisan** [comment]
Yes, resetting to "factory" defaults is a good idea. I'll consider adding such a button.

P.S. I hope you're in fact using [motionEyeOS](https://github.com/ccrisan/motioneyeos), as motionPie is no longer maintained.

**Comment by surak** [comment]
Indeed I am, thanks!
